src/terminal-canvas.h

```diff
@@ -41,6 +41,7 @@ class TerminalCanvas {
     size_t lower_row_pixel_offset_;
     size_t color_fmt_length_;
     std::string buffer_;
+    bool dont_clear_;
```

Comment:
We never use this value outside the constructor, so we don't need to make it an instance variable.